src/lib/LastManStandingUsage.ts

```diff
@@ -0,0 +1,246 @@
+	'{1D} steps out of the safezone too soon and gets killed by the fog.',
+	"{1K} throws a Morrigan Javelin into {2D}'s head.",
+	'{1D} gets 6 Hour logged and dies.',
+	'{1K} catches {2D} off guard and AGS specs a 73.',
+	"{1K} shoots a Dragon arrow into {2D}'s head.",
+	'{1D} cannot handle the circumstances and runs in the fog.',
+	"{1K} bashes {2D}'s head with a Granite maul several times.",
+	'{1K} exsanguinates {2D} with a Ghrazi Rapier.',
+	"{1K} plunges a Dragon dagger in {2D}'s abdomen.",
+	'{1K} sets {2D} on fire with an Infernal cape.',
+	'{1D} falls into a pit and dies.',
+	'{1K} stabs {2D} while their back is turned.',
+	'{1K} severely injures {2D}, but puts them out of their misery.',
+	'{1K} severely injures {2D} and leaves them to die.',
+	"{1K} bashes {2D}'s head in with an Elder maul.",
+	'{1K} pushes {2D} off a cliff during a whip fight.',
+	"{1K} throws a Dragon knife into {2D}'s chest.",
+	'{1D} is unable to convince {2K} to not kill them.',
+	'{1K} convinces {2D} to not kill them, only to kill {2D} instead.',
+	'{1K}, {2}, and {3D} start fighting, but {2} runs away as {1K} kills {3D}.',
+	'{1K} kills {2D} with their own spec weapon.',
+	'{1K} overpowers {2D}, killing them.',
+	'{1K} throws a Chinchompa, killing {2D}.',
+	'{1K} throws a Chinchompa, killing {2D}, and {3D}.',
+	'{1K} throws a Chinchompa, killing {2D}, {3D}, and {4D}.',
+	'{1K} throws a Chinchompa, killing {2D}, {3D}, {4D} and {5D}.',
+	'{1K} kills {2D} as they try to run.',
+	'{1D}, {2D}, {3D}, and {4D} form a suicide pact, and run in the fog.',
+	'{1K} kills {2D} with a Dragon thrownaxe.',
+	'{1K} and {2K} fight {3D} and {4D}. {1K} and {2K} survive.',
+	'{1D} and {2D} fight {3K} and {4K}. {3K} and {4K} survive.',
+	'{1D} attacks {2}, but {3K} protects them, killing {1D}.',
+	'{1K} severely slices {2D} with an Armadyl godsword.',
+	'{1K} strangles {2D} with an Abyssal whip.',
+	'{1K} kills {2D} for their supplies.',
+	'{1K} hurls a Dragon Javelin at {2}, but misses and kills {3D} instead.',
+	"{1K} shoots a poisonous Dragon dart into {2D}'s neck, slowly killing them.",
+	'{1K} stabs {2D} with a Ghrazi rapier.',
+	'{1K} stabs {2D} in the back with a Zuriels Staff.',
+	'{1K}, {2D}, and {3D} get into a fight. {1K} triumphantly kills them both.',
+	'{1D}, {2K}, and {3D} get into a fight. {2K} triumphantly kills them both.',
+	'{1D}, {2D}, and {3K} get into a fight. {3K} triumphantly kills them both.',
+	'{1K} finds {2D} hiding in the Debtor Hideout and kills them.',
+	'{1D} finds {2K} hiding in the Debtor Hideout, but {2K} kills them.',
+	'{1K} kills {2D} with a Dragon scimitar.',
+	'{1K} and {2D} fight for a crate.{1K} strangles {2D} with a whip and runs.',
+	'{1D} and {2K} fight for a crate.{2K} strangles {1D} with a whip and runs.',
+	'{1K} repeatedly stabs {2D} to death with a Ghrazi Rapier.',
+	'{1D} trips over while running from the fog, and is killed by {2K}.',
+	'{1} trips over while running from the fog, {2} picks them up, they run off together.',
```

Comment:
'{1} trips over while running from the fog, {2} picks them up and they run off together.',